packages/faceted-search/src/components/Badges/BadgeSlider/BadgeSlider.component.js

```diff
@@ -0,0 +1,93 @@
+import React, { useMemo } from 'react';
+import PropTypes from 'prop-types';
+import Badge from '@talend/react-components/lib/Badge';
+
+import { BadgeFaceted } from '../BadgeFaceted';
+import { BadgeSliderForm } from './BadgeSliderForm.component';
+import { operatorPropTypes, operatorsPropTypes } from '../../facetedSearch.propTypes';
+
+const createDefaultOperators = t => [
+	{
+		label: t('FACETED_SEARCH_EQUAL', { defaultValue: 'Equal' }),
+		name: 'equals',
+		iconName: 'equal',
+	},
+	{
+		label: t('FACETED_SEARCH_GREATHER_THAN', { defaultValue: 'Greater than' }),
+		name: 'GreaterThan',
+		iconName: 'greater-than',
+	},
+	{
+		label: t('FACETED_SEARCH_LESS_THAN', { defaultValue: 'Less than' }),
+		name: 'LessThan',
+		iconName: 'less-than',
+	},
+];
+
+const BadgeSlider = ({
+	id,
+	initialOperatorOpened,
+	initialValueOpened,
+	label,
+	operator,
+	operators,
+	size,
+	t,
+	value,
+	category,
+	defaultValue = 0,
+	unit = '',
+	...rest
+}) => {
+	const currentOperators = useMemo(() => operators || createDefaultOperators(t), [operators, t]);
+	const currentOperator = operator || currentOperators[0];
+	const badgeTextId = `${id}-badge-slider`;
+	const labelValue = value && `${value}${unit}`;
```

Comment:
I can't take 0 as value and if my value is 0, i should show 0%

Reply:
ow, good catch, thanks